arbnode/transaction_streamer.go

```diff
@@ -670,21 +671,6 @@ func (s *TransactionStreamer) AddFakeInitMessage() error {
 	}})
 }
 
-func (s *TransactionStreamer) isEspressoMode() (bool, error) {
-	config, err := s.exec.GetArbOSConfigAtHeight(0) // Pass 0 to get the ArbOS config at current block height.
```

Comment:
Actually can we remove the `GetArbOSConfigAtHeight` function? What do you think?

Reply:
I was planning to do that in a separate PR with an issue created [here](https://github.com/EspressoSystems/nitro-espresso-integration/issues/393)